Makefile

```diff
@@ -36,6 +38,9 @@ build:
 
 	touch .docker-build
 
+build.py3: override PYTHON_VERSION=3.7.3
+build.py3: build
+
```

Comment:
@jotes I think we should target the `webapp.python3` service here so we can get the `PYTHON_VERSION`